allennlp/modules/conditional_random_field.py

```diff
@@ -364,7 +364,7 @@ def viterbi_tags(
         # Augment transitions matrix with start and end transitions
         start_tag = num_tags
         end_tag = num_tags + 1
-        transitions = torch.Tensor(num_tags + 2, num_tags + 2).fill_(-10000.0)
+        transitions = torch.empty(num_tags + 2, num_tags + 2, device=logits.device).fill_(-10000.0)
```

Comment:
This should use `torch.full()`, so we can create the tensor in one go.

Reply:
It will also be faster!

Reply:
Fixed: https://github.com/allenai/allennlp/pull/5335/commits/84d4e20282ef05a16762cf322683e5b3512db8ab